src/lib/selections.ts

```diff
@@ -536,7 +560,8 @@ export function canSweepSelection(selection: Selections) {
 }
 
 // This accounts for non-geometry selections under "other"
-export type ResolvedSelectionType = [Artifact['type'] | 'other', number]
+export type ResolvedSelectionType = Artifact['type'] | 'other'
```

Comment:
is `other` still needed, not sure it ever get's used now?

Reply:
I used it for an `Axis` selection, which doesn't have an Artifact type that I could see.